api/v1alpha1/perconaservermysql_types.go

```diff
@@ -530,6 +535,18 @@ func (cr *PerconaServerMySQL) CheckNSetDefaults(serverVersion *platform.ServerVe
 		}
 	}
 
+	if cr.Spec.MySQL.ClusterType != ClusterTypeGR && !cr.OrchestratorEnabled() {
```

Comment:
@pooknull one more thing, we should not start the toolkit when the orchestrator is disabled